enhancements/platform/quota-management/README.md

```diff
@@ -0,0 +1,769 @@
+
+```yaml
+apiVersion: quota.datumapis.com/v1alpha1
+kind: ResourceQuotaGrant
+metadata:
+  name: proj-abc
+spec:
+  subjectRef:
+    apiGroup: quota.datumapis.com
+    kind: Project
+    name: proj-abc
+
+  limits:
+  # 1. CPU cores allocated per project / location / instance type
+  - name: compute.datumapis.com/instances/cpu
+    # Dimension labels are the same as the resource name.
+    dimensionLabels:
+      - resources.datumapis.com/project
+      - compute.datumapis.com/location
+      - compute.datumapis.com/instanceType
+    buckets:
+    # Default limit for all locations and instance types across the project.
+    # This is a global limit, and will apply to all locations, since we
+    # don't specify a location dimension label within the bucket selector.
+    - value: "1000"
+      selector: {}
+    # Override for multi-location projects.
+    - value: "300"
+      selector:
+        matchExpressions:
+        - key: compute.datumapis.com/location
+          operator: In
+          values: ["dfw", "lhr", "dls"]
+
+    # Single location and instance type override,
+    # more specific than the global limit and multi-location override.
+    - value: "40"
+      selector:
+        matchLabels:
+          compute.datumapis.com/location: dfw
+          compute.datumapis.com/instanceType: c3.small
+
+  # 2. Memory (GiB) allocated per project / location / instance type
+  - name: compute.datumapis.com/instances/memoryAllocated
+    dimensionLabels:
+      - resources.datumapis.com/project
+      - compute.datumapis.com/location
+      - compute.datumapis.com/instanceType
+    buckets:
+    - value: "4096"
+      selector: {}
+
+    - value: "1024"
+      selector:
+        matchLabels:
+          compute.datumapis.com/location: dfw
+
+  # 3. Instance count per project / location / instance type
+  - name: compute.datumapis.com/instances/count
+    dimensionLabels:
+      - resources.datumapis.com/project
+      - compute.datumapis.com/location
+      - compute.datumapis.com/instanceType
+    buckets:
+    - value: "200"
+      selector: {}
+    - value: "20"
+      selector:
+        matchLabels:
+          instanceType: 
+    - value: "5"
+      selector:
+        matchLabels:
+          compute.datumapis.com/location: dfw
+          compute.datumapis.com/instanceType: c3.small
+
+  # 4. Gateway count per project
+  - name: network.datumapis.com/gateways
+    dimensionLabels:
+      - resources.datumapis.com/project
+    buckets:
+    - value: "15"
+      selector: {}
+
+    - value: "3"
+      selector:
+        matchExpressions:
+        - key: compute.datumapis.com/location
+          operator: Exists
+
+```
+
+
+#### `BillingAccount`
+
+```yaml
+
+```
```

Comment:
We should be able to leave billing accounts out of scope of this document. I would expect to cover that in the billing / metering enhancement.